scripts/02411-Busganda.py

```diff
@@ -0,0 +1,14 @@
+def main():
+    print_details()
+
+def print_details():
+    full_name = ("Busayo Olushola")
+    HNGi7_ID = ("HNG-02411")
+    language = ("python")
+    task = ("stage 2 task")
+    print (f"Hello World, this is {full_name} with HNGi7 ID {HNGi7_ID} using {language} for {task}")

```

Comment:
Please include your email after the word `task` and a space as highlighted [here](https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700)